src/orange/BoundingBoxIO.json.hh

```diff
@@ -7,12 +7,31 @@
 //---------------------------------------------------------------------------//
 #pragma once
 
+#include <limits>
 #include <nlohmann/json.hpp>
 
 #include "corecel/cont/ArrayIO.json.hh"
 
 #include "BoundingBox.hh"
 
+namespace
+{
+void fix_inf(celeritas::Real3* point)
+{
+    constexpr auto max_real = std::numeric_limits<celeritas::real_type>::max();
+    constexpr auto inf = std::numeric_limits<celeritas::real_type>::infinity();
+
+    for (auto axis : range(celeritas::Axis::size_))
+    {
+        auto ax = to_int(axis);
+        if ((*point)[ax] == max_real)
+        {
+            (*point)[ax] = inf;
+        }
```

Comment:
I fixed the `fabs` issue, but now that BoundingBoxIO is templated, I can't split it into a .cc and .hh because the declaration and definition must be in the same translation unit.